truss-chains/truss_chains/utils.py

```diff
@@ -128,6 +130,26 @@ def get_free_port() -> int:
         return port
 
 
+def override_chainlet_to_service_metadata(
+    chainlet_to_service: Mapping[str, definitions.ServiceDescriptor],
+):
+    # Override predict_urls in chainlet_to_service ServiceDescriptors if dynamic_chainlet_config exists
+    dynamic_chainlet_config_str = dynamic_config_resolver.get_dynamic_config_value(
+        definitions.DYNAMIC_CHAINLET_CONFIG_KEY
+    )
+    if dynamic_chainlet_config_str:
+        dynamic_chainlet_config = json.loads(dynamic_chainlet_config_str)
+        for (
+            chainlet_name,
+            service_descriptor,
+        ) in chainlet_to_service.items():
+            if chainlet_name in dynamic_chainlet_config:
```

Comment:
nit: Should we log something, maybe at debug level, if we don't find the `chainlet_name` in the `dynamic_config`? Wondering if this could help us with debugging.